data/templates/centos-ks

```diff
@@ -53,6 +53,13 @@ sudo
 perl
 libselinux-python
 net-tools
+<% if( undefined !== kvm ) { %>
```

Comment:
Since in this PR (https://github.com/RackHD/on-tasks/pull/90/files), the default "kvm" is set to null, here we only judge the "undefined" case, we'd better add a safe-conversion in this function https://github.com/RackHD/on-tasks/blob/master/lib/jobs/install-os.js#L113